Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocListMenuBlockRenderer.cs

```diff
@@ -89,14 +85,13 @@ private void RenderListMenu (BocListRenderingContext renderingContext, string me
           "BocList '{0}': The ListMenu must remain visible if BocList.HasListMenu is evaluates 'true'.",
           renderingContext.Control.ID);
 
-      renderingContext.Writer.AddStyleAttribute (HtmlTextWriterStyle.Width, "100%");
-      renderingContext.Writer.AddStyleAttribute ("margin-bottom", menuBlockItemOffset);
+      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, "listMenuContainer");
```

Comment:
This css class is new, isn't it? If so, please add to the other boclist css classes and rename to `bocListListMenuContainer`

The problem is, the `ListMenuContainer` (to keep the casing symetrical with DropDownMenu) isn't defined on ListMenu but on BocList, so we should prefix it and move it to the other classes.